modules/renter.go

```diff
@@ -192,14 +192,18 @@ func (mrs *MerkleRootSet) UnmarshalJSON(b []byte) error {
 // A RenterContract contains all the metadata necessary to revise or renew a
 // file contract. See `api.RenterContract` for field information.
 type RenterContract struct {
+	HostPublicKey types.SiaPublicKey   `json:"hostpublickey"`
+	ID            types.FileContractID `json:"id"`
+	NetAddress    NetAddress           `json:"netaddress"`
+	SecretKey     crypto.SecretKey     `json:"secretkey"`
+
+	InGoodStanding  bool `json:"ingoodstanding"`  // if false, host is due for replacement
+	UsefulForUpload bool `json:"usefulforupload"` // if false, host cannot be used to upload
```

Comment:
looks like these aren't set to false anywhere. I'm assuming that will be added in later commits.